smac/facade/func_facade.py

```diff
@@ -36,6 +37,8 @@ def fmin_smac(func: callable,
         Maximum number of function evaluations.
     rng : np.random.RandomState, optional
             Random number generator used by SMAC.
+    **kwargs:
+        arguments passed to SMAC facade
```

Comment:
I suggest to not use the word facade here, it's quite confusing when you just want to minimize a function. How about `Arguments passed to the optimizer class.`, ideally with a link.